inspirehep/dojson/hep/fields/bd1xx.py

```diff
@@ -141,6 +145,30 @@ def _get_record(value):
             if x_value and x_value.isdigit():
                 return get_record_ref(x_value, 'authors')
 
+        def _get_contributor_role(value):
+            if not value:
+                return None
+            if not isinstance(value, str):
+                value = set(value)
+                if len(value) is 1:
+                    value = value.pop()
+
+            if value in current_app.config['INSPIRE_LEGACY_ROLES']['editing']:
+                return [
+                    {
+                        'source': 'CRediT',
```

Comment:
I now realize this should have been `'scheme': 'CRediT'`. Because the source is where the data come from, while the schema is the namespace against which the value is written.

Can you amend also the json schema?